packages/ckeditor5-paste-from-office/src/filters/image.ts

```diff
@@ -80,9 +80,18 @@ function findAllShapesIds( documentFragment: ViewDocumentFragment, writer: Upcas
 		const el = value.item as ViewElement;
 		const previousSibling = el.previousSibling;
 		const prevSiblingName = previousSibling && previousSibling.is( 'element' ) ? previousSibling.name : null;
+		// If shape element has 'o:gfxdata' attribute and is not directly before
```

Comment:
Is this comment needed (line 83 and 84)? Same is in 88th line.

Reply:
Good point, I duplicated it accidentally😅